lib/markbind/lib/parser.js

```diff
@@ -35,6 +36,12 @@ const BOILERPLATE_FOLDER_NAME = '_markbind/boilerplates';
  */
 
 function calculateNewBaseUrl(filePath, root, lookUp) {
+  if (!path.isAbsolute) {
+    throw new Error(`Non absolute path given to calculateNewBaseUrl '${filePath}'`);
```

Comment:
- "!path.isAbsolute`(filePath)`"
- "Non`-`absolute"
- "calculateNewBaseUrl`:`"
- Are we throwing because this is a [programmer error](http://2ality.com/2016/03/promise-rejections-vs-exceptions.html#programmer-errors-fail-quickly)? In that case, we should add jsdoc with the [**@throws** tag](http://usejsdoc.org/tags-throws.html).